src/test/kotlin/it/pagopa/wallet/eventdispatcher/service/WalletUsageServiceTest.kt

```diff
@@ -0,0 +1,62 @@
+package it.pagopa.wallet.eventdispatcher.service
+
+import it.pagopa.generated.wallets.model.ClientId
+import it.pagopa.wallet.eventdispatcher.api.WalletsApi
+import it.pagopa.wallet.eventdispatcher.domain.WalletUpdateUsageError
+import java.time.OffsetDateTime
+import java.util.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+import org.mockito.kotlin.*
+import org.springframework.http.ResponseEntity
+import reactor.core.publisher.Mono
+import reactor.kotlin.test.test
+
+class WalletUsageServiceTest {
```

Comment:
maybe those tests can be refactored to user mockWebServer as done [here](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/main/src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/client/NodeClientTest.kt#L34-L223) in order to perform tests with a real http connection being established and use wanted mocked responses (ok, ko, timeout, and so on)